metadata.yaml

```diff
@@ -42,7 +47,7 @@ provides:
 requires:
     ingress:
         interface: ingress
-        limit: 1
+        limit: 2
```

Comment:
Not that this is an issue but why would we need two ingresses ?

Reply:
I increased to 2 because I was testing with Traefik and NGINX at the same time ;-)